network/backend/lnd/lnd.go

```diff
@@ -425,6 +480,10 @@ func New(c *config.Network, logger logging.Logger) (*Backend, error) {
 		logger: logger.With(logging.String("name", c.Name)),
 	}
 
+	if c.PoolCapacity < lndMinPoolCapacity {
```

Comment:
Yeah my pool has bug, technically we want the pool to create a new connection if there is not enough conn.
Your fix can be added temporally the time to fix it